modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/data/provider/RelatedModelsFDSDataProvider.java

```diff
@@ -78,14 +82,47 @@ public List<RelatedModel> getItems(
 		long objectEntryId = ParamUtil.getLong(
 			httpServletRequest, "objectEntryId");
 
+		if (GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-151676")) &&
+			objectDefinition.isSystem()) {
+
+			return TransformUtil.transform(
+				objectRelatedModelsProvider.getRelatedSystemModels(
+					objectEntryId, objectDefinition, objectRelationship),
+				entryMap -> {
+					Long titleFieldId =
+						objectDefinition.getTitleObjectFieldId();
+
+					String titleFieldName =
+						objectDefinition.getPKObjectFieldDBColumnName();
+
+					if (titleFieldId > 0) {
+						ObjectField titleField =
```

Comment:
Would it not be better to rename this variable to objectfField instead?

Reply:
You are right, I will fix this